wgpu/examples/cube/shader.wgsl

```diff
@@ -1,10 +1,10 @@
 struct VertexOutput {
-    @location(0) tex_coord: vec2<f32>;
-    @builtin(position) position: vec4<f32>;
+    @location(0) tex_coord: vec2<f32>,
+    @builtin(position) position: vec4<f32>
```

Comment:
same here and everywhere - please keep the comma

Reply:
I returned back all trailing commas, except structs with one member. Just to show that trailing commas are optional.